website/docs/d/scc_posture_latest_scans.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_latest_scans"
+description: |-
+  Get information about list_latest_scans
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_latest_scans
+
+Review information about the security and compliance center posture latest scans. For more information, about latest scans, see https://test.cloud.ibm.com/docs/security-compliance?topic=security-compliance-results.
```

Comment:
Review information about the security and compliance center posture latest scans. For more information, about latest scans, see https://test.cloud.ibm.com/docs/security-compliance?topic=security-compliance-results.
as 
Review information about the security and compliance center posture latest scans. For more information, about the latest scans, see [viewing evaluation results](https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-results).

Note: please do provide the hyperlink using []() syntax, and should not provide stage docs link for the registry docs.